tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,338 @@
+"""Tests for the Incremental Update feature"""
+
+from datetime import date, timedelta
+from fauxfactory import gen_alpha
+from nailgun import entity_mixins
+from nailgun.entities import (
```

Comment:
The common usage for entities are like:

```python
from nailgun import entities

entities.Something().create()
```

This is not a blocker but it should be better to stick with the common usage.